frontend/src/components/core/Button.tsx

```diff
@@ -94,7 +95,15 @@ export const ProblemIOTypeButton = styled(DefaultButton)<ProblemIOTypeButtonProp
 
 export const SmallButton = styled(DefaultButton)`
   color: ${({ theme }) => theme.colors.white};
-  background-color: ${({ theme }) => theme.colors.blue};
-  padding: 0.5rem 1rem;
+  background: ${({ theme }) => theme.colors.gradients.blue};
+  font-size: ${({ theme }) => theme.fontSize.xMediumSmall};
+  padding: 0;
   margin: 0.4rem;
+  width: 5.8rem;
+  height: 2.1rem;
```

Comment:
Thoughts on rounding these numbers slightly? Not sure how you got these decimals, maybe from the Figma, but it might still look ok just rounded to the nearest 0.5 value.

Reply:
Haha yeah I might have overengineered this - as a compromise I can change it to 6 and 2.25 if that helps (6-2 and 6-2.5 I felt were just a tad bit too long/tall respectively)

Reply:
That works by me haha, also this is just a nit so if you feel it works best with these numbers, no problem.